platform/working-with-vsp/policies-work-norms/deployment-policy.md

```diff
@@ -42,31 +42,21 @@ The holiday release freeze is in effect during the following dates:
 
 # Requesting out-of-band deploys
 
-If there is a _critical issue_ that needs to be resolved outside the automated deployment schedule, explicit permission must be granted for an out-of-band deploy.
+Until better isolation is in place within the codebase and infrastructure, the platform will not entertain requests for deployments outside of the automated deployment schedule. Thus, VFS teams need to make sure they have thoroughly QA'd and otherwise tested the code they're checking in, as updates will not be possible until the following day.
 
-See [Resolving critical issues](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/resolving-critical-issues.md) for recommended approaches to issue resolution (i.e. when to revert, when to fix forward).
-
-For every out-of-band deploy requested, VSP team will expect a follow-up [postmortem](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/Postmortems) from the requesting team, explaining the context that led to the problem and proposing follow-up actions to prevent similar future problems.
-
-## Is my issue critical?
+However, if there is an _outage_ or a _security issue_ that requires immediate resolution and cannot wait for the daily deploy, it is appropriate to escalate to the Platform Crew Leads for a decision on whether to pursue an out-of-band deploy. 
 
-Examples of _critical issues_ include:
-* A bug that’s preventing a significant number of Veterans from accessing a feature
-* A bug creating a non-trivial deviation from expected functionality
-* A 508/accessibility failure of severity level **0** (_Showstopper_) or **1** (_Critical_) ([severity rubric](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md))
+The VFS team whose code triggered the outage or security issue will need to publish a follow-up [postmortem](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/tree/master/Postmortems) in the oncall channel, with the aid of the others involved in resolution.
 
-Examples of _non-critical issues_ include:
-* Incorrect text or visual formatting that does not impede the feature from working
-* Any code for features not yet released to Veterans
-* Just wanting to get code out sooner
-
-When in doubt on whether an issue is _critical_ enough for out-of-band deployment, DEPO leadership for VSP will make the deciding call.
+See [Resolving critical issues](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/resolving-critical-issues.md) for recommended approaches to issue resolution (i.e. when to revert, when to fix forward).
```

Comment:
Does this mean out-of-band deployments only occur for "critical" issues? When should [Resolving critical issues](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/resolving-critical-issues.md) be followed vs "Step-by-step process for out-of-band deployment approval" below?